packages/airnode-node/src/evm/requests/event-logs.ts

```diff
@@ -33,12 +34,13 @@ export function parseAirnodeRrpLog<T = { readonly args: any }>(log: ethers.provi
 }
 
 export async function fetch(options: FetchOptions): Promise<EVMEventLog[]> {
-  // Protect against a potential negative fromBlock value
+  // Protect against a potential negative fromBlock and toBlock value
```

Comment:
This sentence isn't syntactically correct anymore. Maybe revert to what it was and add a new comment above `const toBlock`. Something along the lines of `// toBlock should always be >= fromBlock`

Reply:
I was thinking more along the lines of `Protect against a potential toBlock value that is lower than fromBlock value`  but that also works.